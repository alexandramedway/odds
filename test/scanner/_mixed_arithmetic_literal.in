10 + 10.34 ** 0.5 / 10-20.0
