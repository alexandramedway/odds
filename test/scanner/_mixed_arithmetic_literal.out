NUM_LITERAL
PLUS
NUM_LITERAL
POWER
NUM_LITERAL
DIVIDE
NUM_LITERAL
MINUS
NUM_LITERAL
